cmd/network-tracer/tracer.go

```diff
@@ -91,3 +93,34 @@ func (nt *NetworkTracer) Close() {
 	nt.conn.Stop()
 	nt.tracer.Stop()
 }
+
+// occasionally the eBPF module will miss tcp_close calls and leave TCP
+// connections in an open state. This method filters out TCP connections
+// associated with processes that aren't alive
+func filterDeadConnections(conns *tracer.Connections) {
+	if conns == nil {
+		return
+	}
+
+	var filteredConnections []tracer.ConnectionStats
+
+	deadProcs := 0
+	for _, conn := range conns.Conns {
+		if conn.Type == tracer.TCP && isDeadPID(conn.Pid) {
+			deadProcs++
+		} else {
+			filteredConnections = append(filteredConnections, conn)
+		}
+	}
+	log.Infof("there were %d dead connections in connection capture")
```

Comment:
Should this be at the debug level? Apart from startup, it doesn't look like we log at info anywhere else here.